README.md

```diff
@@ -1,3 +1,80 @@
-# The Nivenly Covenant
+# The Nivenly Covenant 1.0
+
+Include a published and publicly accessible statement you to agree and commit to the following terms. 
+
+``` 
+<Entity,Person(s),Authors,Organization(s)> herby agree an unequivocally commit to the fullest
+extent of our understanding, our intentions, and our goals to The Nivenly Covenant 1.0
+```
+
+### 1. Sustainability
+
+We agree that sustainability is our critical priority, and is necessary for the longevity of future generations of stewards and technologists.
+
+We agree to prioritize the sustainability, mental, and physical health, well-being, overall working condition, and operational excellence for the people working on the project, the source code of the project, and the day-to-day operations of the project.
+
+We understand that the longevity and success of this project is intended to be given to another generation of stewards long after the original authors have completed their work on the project. We aim to leave this project in better condition than we inherited it.
+
+We understand that "right" is more important than "right now", and that future generations are more important than the needs of today.
+
+Given the decision between profit and sustainability, we unequivocally chose sustainability.
+
+### 2. Ownership
+
+We agree to examine our relationship with ownership, and understand the consequences of our decisions with regard to ownership.
+
+#### 2.a Equal Ownership
+
+We acknowledge that an imbalance in ownership creates an imbalance in the health of our people and our works.
+
+We understand that private ownership is a great responsibility that requires special attention to detail in order to operate sustainably. We have considered releasing our work to the public domain and have determined that private ownership is a more appropriate ownership model for our work.
+
+If we chose to instill a concept of ownership for the project, we agree to share the ownership equally amongst those that maintain and contribute to the project.
+
+#### 2.b Stewardship
+
+As an alternative to private ownership, we have considered releasing our work to the public domain.
+
+We understand that the public domain is a dedicated space, where there is no concept of private ownership. In the event we release our project to the public domain, we agree to serve needs of the general public and act as responsible software stewards. We understand that we have an obligation to the public conservation of our work. As a steward we understand we are in position of burden; and not that of authority.
```

Comment:
can we require a "public domain" license like CC0 or similar? something well-recognised.